agent-inject/agent/annotations_test.go

```diff
@@ -218,6 +219,66 @@ func TestSecretTemplateAnnotations(t *testing.T) {
 	}
 }
 
+func TestTemplateShortcuts(t *testing.T) {
+	tests := []struct {
+		name            string
+		annotations     map[string]string
+		expectedSecrets map[string]Secret
+	}{
+		{
+			"valid inject-token-only",
+			map[string]string{
+				"vault.hashicorp.com/agent-inject-token-only": "true",
```

Comment:
Nit: use the annotation variable.

Reply:
Changed in d9dcbcddb2d915921112e0e98258fec3052bce40